runtime/oti/j9dump.h

```diff
@@ -85,7 +85,8 @@ typedef struct J9RASdumpAgent {
 #define J9RAS_DUMP_ON_OBJECT_ALLOCATION_BIT  21
 #define J9RAS_DUMP_ON_CORRUPT_CACHE_BIT  22
 #define J9RAS_DUMP_ON_EXCESSIVE_GC_BIT  23
-#define J9RAS_DUMP_HOOK_TABLE_SIZE  24  /* 1+ the last _BIT */
+#define J9RAS_DUMP_ON_USER2_SIGNAL_BIT  24
+#define J9RAS_DUMP_HOOK_TABLE_SIZE  25  /* 1+ the last _BIT */
```

Comment:
FYI, I've noticed that none of these `*_BIT` macros are used anywhere. I plan to remove them all in an upcoming pull request.